docs/migration.md

```diff
@@ -4,6 +4,13 @@
 
 ### Breaking Changes
 
+#### All Packages
+
+- Garden's React packages now require a minimum peer dependency of
+  `styled-components` at `v6.0.0`.
+  - Any custom styled components should follow the styled components v6
+    [migration requirements](https://styled-components.com/docs/faqs#what-do-i-need-to-do-to-migrate-to-v6).
```

Comment:
This is not correct. When v9 ships, the minimum for `styled-components` will be `v5.1.0`. After v9, we'll chip away at renaming transient styled props, and allow a `^5.1.0 || ^6.00` dependency range on a package-by-package basis. But there is no need to spell that out here. See sequence doc for details.

Reply:
I misinterpreted the sequence. Sounds good.

Reply:
Both the plan sequence and ticket say the v5 range is `5.0.0` for the v5 range in `react-theming` - probably just a typo. I'll update the branch & ticket to say `5.1.0`.

Reply:
Yeah, we need `5.1.0` in order to make the jump to tran$ient props before they are forced in `6.0.0`.